ecommerce/extensions/partner/strategy.py

```diff
@@ -33,8 +36,25 @@ def availability_policy(self, product, stockrecord):
         return availability.Unavailable()
 
 
+class TaxStrategyMixin(strategy.FixedRateTax):
+    """
+    Tax strategy for e-commerce pricing.
+
+    Defaults to 0% tax, but allows setting a fixed tax rate through Django settings.
+    """
+    rate = D(settings.TAX_RATE)
+
+    def get_rate(self, product, stockrecord):
+        """
+        Only return tax rate if it's enabled in settings, otherwise return 0.
+        """
+        if not settings.ENABLE_TAX_RATE:
+            return D('0')
+        return self.rate
+
+
 class DefaultStrategy(strategy.UseFirstStockRecord, CourseSeatAvailabilityPolicyMixin,
-                      strategy.NoTax, strategy.Structured):
+                      TaxStrategyMixin, strategy.Structured):
```

Comment:
@giovannicimolin, wouldn't it be possible to define the mixin and a new strategy class in PayTabs plugin app and use it as the strategy for PayTabs, without requiring this code drift?

Reply:
@lgp171188 No, in that case, the value including tax would not be displayed to the user in the basket (before anything related to the payment provider happens).
This is one of our requirements, so I'm implementing this to give proper tax support throughout the application.